contracts/base/EtherCollateralLoans.sol

```diff
@@ -114,6 +112,8 @@ contract EtherCollateralLoans is LoansBase {
             settingsAddress,
             marketsAddress
         );
+
+        collateralToken = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
```

Comment:
Did we remove type and scope?

Reply:
It is defined in `LoansBase.sol`. Contracts that are **upgradable** should not be instantiated outside the `initialize` function except for **constant**s.